scripts/prepublish

```diff
@@ -1,12 +1,23 @@
-#!/bin/bash -xe
+#!/bin/bash -e
+
 rm -rf ./dist/
 yarn extract_abi
 yarn generate
+
 mkdir -p helpers/compiled/
 
 CONTRACTS="RelayHub StakeManager Penalizer TrustedForwarder TestPaymasterEverythingAccepted"
 BUILDFOLDER=./build/contracts
-OUTFOLDER=./helpers/compiled/
-for c in $CONTRACTS ; do cp $BUILDFOLDER/$c.json $OUTFOLDER ; done
+OUTFOLDER=./helpers/compiled
+for c in $CONTRACTS; do
+  echo copy $OUTFOLDER/$c.json
+  cp $BUILDFOLDER/$c.json $OUTFOLDER
+done
+
+
+for c in GsnDevProvider RelayProvider RelayClient; do
```

Comment:
Two questions:
1. Why do we need these files for RelayProvider and RelayClient, they are exported as index.js, I thought only GsnDevProvider creates a problem for us?
2. Could you please add these files to .gitignore as well?

Reply:
Technically, you can "import all" and its OK. I wanted to allow "surgical import" of inner contracts, without requiring to know the entire src path.